components/Core/Title/index.tsx

```diff
@@ -0,0 +1,9 @@
+interface TitleProps {
```

Comment:
import react, this does not work I think otherwise

Reply:
NextJs imports react automatically in JSX files. So if only JSX notation is used, you don't need to import it.